core/src/test/java/io/questdb/test/griffin/engine/functions/activity/QueryActivityFunctionFactoryTest.java

```diff
@@ -253,8 +243,8 @@ public String getPrincipal() {
 
     private static class UserContext extends ReadOnlySecurityContext {
         @Override
-        public void authorizeAdminAction() {
-            throw new CairoException();
+        public void authorizeQueryAdmin() {
+            throw CairoException.authorization().put("Access denied for ").put(getPrincipal()).put(" [QUERY ADMIN]");
```

Comment:
lets rename QUERY ADMIN to "SQL Engine Admin"

Reply:
renamed it